src/controllers/taskController.js

```diff
@@ -562,7 +563,8 @@ const taskController = function (Task) {
       res.status(400).send({ error: 'Task Name, Active status, Task Number are mandatory fields' });
       return;
     } */
-
+    /* Remove Cache If Data Exist */
+    cache.removeCache(req.params.id);
```

Comment:
just to make absolutely sure it is absolutely unique may want to do ("task-" + req.params.id)